### DISCUSSION TITLE
可以增加TextBundle格式支持

### DISCUSSION THREAD

**Discussion by lrkkevin** [vuepress-theme-hope/vuepress-theme-hope#3988] [Q&A] [answered]
可以增加TextBundle格式支持

**Comment by Mister-Hope** [comment, marked as answer]
This should be asked in vuepress/core repo, not here.

Additionally, I don't think VuePress should support this format.

It's possible for you to add a pre-script that unzip all the `*.textbundle` files to a temp folder, and let vuepress serve that temp folder.
